examples/EagerMode/QuickStart/README.md

```diff
@@ -0,0 +1,32 @@
+## Dygraph Quick Start
+Train an agent with PARL to solve the CartPole problem, a classical benchmark in RL. Dygraph version of [QuickStart][origin]
+
+## How to use
+### Dependencies:
+
++ [paddlepaddle>=1.5.1](https://github.com/PaddlePaddle/Paddle)
++ [parl](https://github.com/PaddlePaddle/PARL)
++ gym
+
+### Start Training:
+```
+# Install dependencies
+pip install paddlepaddle  
+# Or use Cuda: pip install paddlepaddle-gpu
+
+pip install gym
+git clone https://github.com/PaddlePaddle/PARL.git
+cd PARL
+pip install .
+
+# Train model
+cd examples/EagerMode/QuickStart/
+python train.py  
+```
+
+### Expected Result
+<img src="performance.gif" width = "300" height ="200" alt="result"/>
```

Comment:
can we add this figure by referencing to the figure used in the quickstart folder, instead of making a copy of it.